packages/components/nodes/chains/LLMChain/LLMChain.ts

```diff
@@ -144,14 +153,25 @@ const runPrediction = async (
     const isStreaming = options.socketIO && options.socketIOClientId
     const socketIO = isStreaming ? options.socketIO : undefined
     const socketIOClientId = isStreaming ? options.socketIOClientId : ''
-
+    const moderations = nodeData.inputs?.inputModeration as Moderation[]
     /**
      * Apply string transformation to reverse converted special chars:
      * FROM: { "value": "hello i am benFLOWISE_NEWLINEFLOWISE_NEWLINEFLOWISE_TABhow are you?" }
      * TO: { "value": "hello i am ben\n\n\thow are you?" }
      */
     const promptValues = handleEscapeCharacters(promptValuesRaw, true)
 
+    if (moderations && moderations.length > 0) {
+        try {
+            // Use the output of the moderation chain as input for the LLM chain
+            input = await checkInputs(moderations, chain.llm, input)
+        } catch (e) {
+            await new Promise((resolve) => setTimeout(resolve, 500))
```

Comment:
adding slight delay because when the response return too fast, the order of the chat message returned became opposite:
![Screenshot 2023-11-22 165835](https://github.com/FlowiseAI/Flowise/assets/26460777/135c8c83-9758-4d13-80df-43d084ebe34e)